assets/monitoring/grafana/v1alpha1/dashboards.cm.yaml

```diff
@@ -4602,15 +4602,15 @@ data:
                             ]
                         },
                         "datasource": "prometheus",
-                        "definition": "query_result(count(up{job=\"scylla\"}) by (dc))",
+                        "definition": "query_result(count(up{job=~\"$cluster|$^\"}) by (dc))",
```

Comment:
can you pls explain the syntax beyond `job=~'${cluster}'` to me? in particular why the "or" (`|`) and what `$^` means here?

Reply:
I shamelessly copied it over from the other queries - and I suppose you did the same, as it originates from scylla-monitoring dashboards. I actually tried to find something on it in the docs with no success. This essentially substitutes the variables and then matches a regex, so for a cluster named `simple-cluster` you end up matching e.g. `simple-cluster|$^`, but I don't see how that's any different than matching just `simple-cluster`, with or without the regex. @amnonh would you be able to explain? Is that some grafana/prom query magic?

Reply:
I am fine deferring that one and wait for @amnonh's input. Please file an issue to track this and we can merge this at least.

Reply:
Opened https://github.com/scylladb/scylla-operator/issues/1209